nautobot/dcim/utils.py

```diff
@@ -65,6 +65,11 @@ def cable_status_color_css(record):
         return CABLE_STATUS_TO_CSS_CLASS.get(status_color, "")
 
 
+def get_network_driver_mapping_for_platform(platform):
+    tool_names = get_network_driver_mapping_tool_names()
+    return {tool_name: platform.network_driver_mappings.get(tool_name) for tool_name in tool_names}
```

Comment:
We need to check if the platform is `None` here and if so return an empty dictionary {}.
In addition, It would be good to add a docstring here and rename the function to a more appropriate name. I just hacked this together in the meeting in a hurry.